trikotFoundation/src/nativeMain/kotlin/com/mirego/trikot/foundation/date/Date.kt

```diff
@@ -27,17 +31,45 @@ actual class Date(val nsDate: NSDate) {
                 return Date(NSDate())
             }
 
+        @ExperimentalUnsignedTypes
         actual fun fromISO8601(isoDate: String): Date {
-            return Date(
-                NSISO8601DateFormatter().dateFromString(
-                    isoDate
-                ) ?: NSDate()
-            )
+            val systemVersion = UIDevice.currentDevice.systemVersion.toFloat()
+            return when {
+                floor(systemVersion) >= 11 ->
```

Comment:
I'm not sure but is this code also used for tvOS? If so it shouldn't be a big deal since [NSISO8601DateFormatWithFractionalSeconds](https://developer.apple.com/documentation/foundation/nsiso8601dateformatoptions/nsiso8601dateformatwithfractionalseconds?language=objc) is available for iOS 11+ and tvOS 11+ but maybe someone can confirm?

Reply:
make sens

Reply:
@CharlesMcD As this code is written in the `nativeMain` scope, it **will** be available on tvOS by default as the gradle build task depends on nativeMain. See https://github.com/mirego/trikot.foundation/blob/620be21afc7c81cf7537e513627103cedb55df84/trikotFoundation/build.gradle#L119

Reply:
Ok it's what I thought but I don't think it's going to be a problem. Do you see a potential bug with that? @marcantoinefortier

Reply:
I don't see any issues with that. Should work as expected on both iOS and tvOS. 🙂